contracts/Multicall.cairo

```diff
@@ -1,49 +1,32 @@
 %lang starknet
 
-from starkware.starknet.common.syscalls import call_contract
+from starkware.starknet.common.syscalls import call_contract, get_block_number
 from starkware.cairo.common.alloc import alloc
 from starkware.cairo.common.memcpy import memcpy
 
-
 #########################################################################
 # The Multicall contract can call an array of view methods on different
-# contracts and return the aggregate responses as an array.
+# contracts and return the aggregate response as an array.
 # E.g.
 # Input: [to_1, selector_1, data_1_len, data_1, ..., to_N, selector_N, data_N_len, data_N]
 # Output: [result_1 + .... + result_N]
 #########################################################################
 
 @view
-func multicall{
-        syscall_ptr: felt*,
-        range_check_ptr
-    } (
-        calls_len: felt,
-        calls: felt*
-    ) -> (result_len: felt, result: felt*):
+func aggregate{syscall_ptr : felt*, range_check_ptr}(calls_len : felt, calls : felt*) -> (
```

Comment:
Can you keep the existing formatting to be consistent with the other contracts of the repo?